demo/demo-tekton.sh

```diff
@@ -221,17 +239,33 @@ printf "Templates: "
 oc delete --all template.template.openshift.io -n "${app_namespace}" 2> /dev/null || echo "...done"
 printf "RBAC Authorization: "
 oc delete "clusterrolebinding.rbac.authorization.k8s.io/pipeline-role-binding-${app_namespace}" 2> /dev/null || echo "...done"
-pv="$(oc get pv | grep "${app_name}" | cut -d " " -f 1)"
-pvc="$(oc get pvc -n "${app_name}" | cut -d " " -f 1 | tail -n 1)"
-printf "Remove PV's and PVC's associated w/ the tekton demo: %s and %s\n" "$pv" "$pvc" 
-oc delete pv "$pv" --grace-period=0 --wait=false || true
-oc delete pvc "$pvc" -n "${app_namespace}" --grace-period=0 --wait=false || true
-oc patch pvc "$pvc" -p '{"metadata":{"finalizers":null}}' -n "${app_namespace}" || true
+if oc get pv | grep "${app_name}"; then
+    pv="$(oc get pv | grep "${app_name}" | cut -d " " -f 1)"
+    printf "Remove PV's associated w/ the tekton demo: %s\n" "$pv"
+    oc delete pv "$pv" --grace-period=0 --wait=false || true
+else
+    echo "No PV's to remove"
```

Comment:
Is it warning or error? If second should the script exit with error ?

Reply:
I think it is just a clean up before start, I would not exit. If we exit, first time someone runs the scripts, it will always fail, no?

Reply:
There is a difference between PV's and PVC's  and they both need to be cleared.   I'll update this.